Cargo.toml

```diff
@@ -5,6 +5,12 @@ name = "{{project-name}}"
 edition = "2018"
 version = "0.1.0"
 
+[lib]
+test = false
```

Comment:
is this needed when `src/lib.rs` is missing? if it is don't you also need `doctest = false`?

Reply:
ah, wait this is the main Cargo.toml so maybe it is needed

Reply:
Ah, no, this is only needed when you want `cargo test --all` to work, but for that we also need to set `test = false` for every binary, so this doesn't do anything useful for us.